src/commands/math/advancedSymbols.ts

```diff
@@ -737,3 +737,227 @@ LatexCmds.measuredangle = bindVanillaSymbol(
   '&#8737;',
   'measured angle'
 );
+
+//AL-1830: added additional keys for copy-pasting unicode characters
+LatexCmds['\\'] = LatexCmds['backslash'];
```

Comment:
Unicode backslash is a reserved character in latex, double backslash means newline.
```suggestion
```